src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -5,27 +5,28 @@
             <i *ngIf="showLinkComponent" class="fa fa-angle-down"></i>
             <i *ngIf="!showLinkComponent" class="fa fa-angle-up"></i>
         </span>
-        <div class="link-header-nav" 
+        <ul class="link-header-nav-list">
+            <li class="link-header-nav"  
```

Comment:
Please maintain the consistency of indentation in the file. It's becoming a mix of 2 vs 4 space indent.